std/range/package.d

```diff
@@ -3877,7 +3894,6 @@ pure unittest
     assert(b == [ "c", "b", "a" ]);
```

Comment:
This is the already existing unittest example, but yeah it looks at lot nicer :)